functions/graphql/index.js

```diff
@@ -24,14 +25,54 @@ function configureServer() {
   //use the cors middleware
   app.use(cors());
 
+  // Set up for the file middleware
+  app.use((req, res, next) => {
+    if (req.body instanceof Buffer) {
+      req.body = {};
+    }
+    next();
+  });
+
+  app.use(fileMiddleware);
+  app.use((req, res, next) => {
+    // Map the files onto the query.
+    if (req.files) {
+      const ops = JSON.parse(req.body.operations);
+      const maps = JSON.parse(req.body.map);
+      const filePaths = Object.entries(maps).reduce(
+        (prev, [key, pathArr]) =>
+          prev.concat([[pathArr, req.files.find(f => f.fieldname === key)]]),
+        []
+      );
+      const mapped = filePaths.reduce(
+        (prev, [paths, file]) => ({
+          ...prev,
+          ...paths.reduce(
+            (p, path) => ({
+              ...p,
+              ...path
+                .split(".")
+                .reverse()
+                .reduce((pp, next) => ({ [next]: pp }), file)
+            }),
+            {}
+          )
+        }),
+        {}
+      );
+      //  "variables.picture"
+      req.body = {
+        ...ops,
+        variables: { ...ops.variables, ...mapped.variables }
+      };
+    }
+    next();
+  });
   const server = new ApolloServer({
     schema,
     engine: process.env.ENGINE_API_KEY,
     tracing: process.env.NODE_ENV !== "production",
-    uploads: {
-      maxFileSize: 10000000, // 10 MB
-      maxFiles: 20
-    },
+    uploads: false,
```

Comment:
Is this disabling their ability to upload files?

Reply:
This does not. Rather, I handle uploading files using the middleware instead of the built-in stuff. See the above answer.